core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -154,6 +166,20 @@ private void addBlockId(long blockId, long sizeBytes, long accessTimeMs, boolean
   }
 
   /**
+   * Attenuate the reference Frequency of each block by halving the value. This should be executed
+   * periodically.
+   */
+  public void attenuateBlockReferenceFrequency() {
```

Comment:
This should be executed periodically. I expose it to be called by worker's heartbeat.